docs/examples/tutorial/rectangle_cpp/setup.py

```diff
@@ -0,0 +1,9 @@
+from distutils.core import setup, Extension
+from Cython.Build import cythonize
+
+setup(ext_modules = cythonize(Extension(
+           "rect",                                # the extension name
+           sources=["rect.pyx", "Rectangle.cpp"], # the Cython source and
```

Comment:
This is redundant because the configuration already exists in the source file. I think it's better to keep it there, so that this simplifies to `cythonize("rect.pyx")`.